ModelUploader/Program.cs

```diff
@@ -125,6 +137,17 @@ static void Main(string[] args)
 
         }
 
+        private static void PopulateModelFileMap(string modelPath)
+        {
+            // Go over directories
+            EnumerationOptions options = new EnumerationOptions() { RecurseSubdirectories = true };
+            foreach (string file in Directory.EnumerateFiles(modelPath, "*.json", options))
+            {
+                BasicModelInterface modelInterface = JsonSerializer.Deserialize<BasicModelInterface>(File.ReadAllText(file));
```

Comment:
I have the same comment as Karl, I didn't fix it in my previous PR.
Why deserialize into a base object (that then needs to be cast to JsonElement -- could that fail?) rather than parse directly into JsonDocument?

Reply:
Probably better to deserialize into JsonDocument. But a little bit more complicated to get the needed info. Next time.